### PR TITLE
Create menupage

### PR DESCRIPTION
**Pull request by poongodimohan**

### CODE REVIEW

File: src/pages/menus/MenuPageSlicker.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    />
+                    <Box>
+                        <Fade left>
+                            <Typography
+                                variant="h4"
+                                sx={{
+                                    position: "absolute",
+                                     top: "100%",
+                                     left: "35%",
+                                    // justifyContent: 'center',
```
Comment (kiruthikaSukumar) [inline src/pages/menus/MenuPageSlicker.tsx]: remove this

File: src/pages/menus/MenuPageSlicker.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                                    alignItems: 'center',
+                                    transform: "translateY(-200%)",
+                                    color: "#fff", // Adjust the text color
+                                    fontSize: "6rem", // Adjust the text font size
+
+                                    '@media screen and (max-width: 1024px)': {
+                                        // Adjust styles for iPad
```
Comment (kiruthikaSukumar) [inline src/pages/menus/MenuPageSlicker.tsx]: remove this line

File: src/pages/menus/Menus.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            >
+                <MenuPageSlicker />
+            </Box>
+
+            <Box
+                sx={{
+                    position: 'relative',
+                    zIndex: 1, // Ensure the content is above the background image
+                    mb: 3,
+                    // pt: '150px
```
Comment (kiruthikaSukumar) [inline src/pages/menus/Menus.tsx]: remove this

Comment (poongodimohan) [reply src/pages/menus/Menus.tsx]: removed

**Pull request merged**